GTBitsOfGood/canine-assistants
# Disable Loading Buttons

## Disable Loading Buttons

Issue Number(s): #200

Disables buttons that involve making an AJAX request and/or going to a new page until that action is complete. Also adds a hover state to all buttons.

### Checklist


### How To Test

Ensure that all buttons that 'wait' to complete an action have a waiting state, and also that all primary/secondary/tertiary buttons in general have a hover state.
